lib/mods/replace-get-set.js

```diff
@@ -25,25 +25,60 @@ function isDataProperty(type, propertyName) {
 }
 
 /**
- * @param {Node} node - the current ts-morph AST Node
+ * @param {Node} node - the ts-morph AST Node to evaluate
  *
- * @return {boolean} whether the node is a global get
+ * @return {boolean} whether the node is a global property access
  */
-function isGlobalGet(node) {
+function isGlobalPropertyAccess(node) {
   return nodeTypes.isPropertyAccessExpression(node) &&
     !nodeTypes.isCallExpression(node.getParent()) &&
     isGlobal(node.getExpression());
 }
 
 /**
- * @param {Node} node - the current ts-morph AST Node
+ * @param {Node} node - the ts-morph AST Node to evaluate
  *
  * @return {boolean} whether the node is a global set
  */
-function isGlobalSet(node) {
+function isSet(node) {
   return nodeTypes.isBinaryExpression(node) &&
-    node.getOperatorToken().getText() === '=' &&
-    isGlobalGet(node.getLeft());
+    node.getOperatorToken().getText() === '=';
+}
+
+/**
+ * @param {Node} node - the ts-morph AST Node to evaluate
```

Comment:
Can you state what type of node it is? I think it's Identifier?

Reply:
I don't think we know that for certain, it will only pass through if it's an Identifier, but as of right now it gets every node. Looking at this with that perspective though, I'm going to add a check in here for Identifiers.

Reply:
I am pretty sure it is an identifier. Can you give an example (e.g. by console.loging it) where it is not?

Reply:
Attached is the output of some of this, because it's checking every node. What I can do is add a first filter checking if it's an identifier, but unless I did that in both if statements using `isWindowGlobal` then it's still gonna take in `Node`

```    
    const a = window.closed;
    const a = window.closed
    a = window.closed
    a
    window.status = 'open';
    status = 'closed';
    status = 'closed'
    status
    const s = status;
    const s = status
    s = status
    s
    status
    const bat = navigator.getBattery();
    const bat = navigator.getBattery()
    bat = navigator.getBattery()
    bat
    navigator.getBattery()
    navigator.getBattery
    navigator
    getBattery
    Window_alert(status);
    Window_alert(status)
    Window_alert
    status
```

Reply:
I see. Well yeah, the rest of the code in this function doesn't make a lot of sense for non-identifiers. So maybe just add `if (!identifier) { return; }` at the top of this function?